packages/core/src/core.ts

```diff
@@ -41,16 +41,11 @@ export function exportVariable(name: string, val: string): void {
 }
 
 /**
- * exports the variable and registers a secret which will get masked from logs
- * @param name the name of the variable to set
+ * Registers a secret which will get masked from logs
  * @param val value of the secret
```

Comment:
`secret` instead of `val`, or the other way, make sure the method and comment lines up. 😆

Reply:
will do